### ISSUE TITLE
Open Orbiter Release lacks debug libs.

### ISSUE THREAD

**Issue by jarmonik** [orbitersim/orbiter#493]
I noticed that current Open Orbiter RC1 lacks degub libraries such as orbiterd.lib, orbitersdkd.lib, Lua/*.lib etc. Currently the idea was to manually build a debug version of OO, rename the libs and add to release package. Making this automatic looks quite complex.

**Comment by DarkWanderer** [comment]
I think it may be possible to do

What is the expectation? Should OrbiterSDK be built in Debug mode and `.lib` files placed into same folders as release ones? Is there a list of libraries?

**Comment by jarmonik** [comment]
Yes, the Debug libs should be in the same place with the Release libs with 'd' added to the name. There was some discussion about having Debug libs in a different location /lib/debug/ but the 'd' addition was considered to be more standard method. I believe all the libs in /lib/ would be required, meaning all 5 of them.
